app/controllers/spree/orders_controller.rb

```diff
@@ -27,6 +27,7 @@ class OrdersController < ::BaseController
     def show
       @order = Spree::Order.find_by!(number: params[:id])
       ProcessPaymentIntent.new(params["payment_intent"], @order).call!
+      @order.reload
     end
```

Comment:
I'm missing controller tests for this new payment-intent behavior.

Reply:
05e762ff2151c0fd5406f000a8ba3795f28fe7d3